apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/styles/starter-page-templates-editor.scss

```diff
@@ -69,7 +69,26 @@ body.admin-bar:not( .is-fullscreen-mode ) .page-template-modal-screen-overlay {
 
 // Close button
 .page-template-modal .components-modal__header .components-button {
-	left: 0;
+	left: -24px; // Make up for 24px modal padding.
+	margin: 9px 10px;
+	padding: 4px;
+
+	.dashicon {
+		width: 28px;
+		height: 28px;
+	}
+}
+
+.page-template-modal .components-modal__header {
+	position: relative;
+
+	&::after {
```

Comment:
It would be great to add a comment here detailing what this is doing. It will help the next developer who comes to this code.